apps/docs/src/examples/renderers/CSS2DRenderer/CssObject.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import { T } from '@threlte/core'
+  import { CSS2DObject } from 'three/examples/jsm/renderers/CSS2DRenderer'
+
+  export let pointerEvents = false
+
+  let element
+
+  const cssobject = (node) => {
+    element = node
+    return {
+      destroy() { element = undefined }
+    }
+  }
+</script>
+
+<div
+  use:cssobject
+  style:pointer-events={pointerEvents ? 'auto' : 'none !important'}
+  style:will-change="transform"
+>
+  <slot />
+</div>
+
+{#if element}
+  <T {...$$restProps} is={CSS2DObject} args={[element]} let:ref>
+    <slot name="three" {ref} />
```

Comment:
It would be neat if we could demonstrate the use of this slot without overcrowding the scene, but I don't have a good idea of how.

Reply:
I'll be honest - I don't know what this slot is for or how to use it. I copied the code in from the original example and didn't much think about it.

Reply:
I've removed it from the example, since it doesn't seem necessary to get everything working.